BetterCallSodium/External/MonoPInvokeCallbackAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma warning disable IDE0052
+// ReSharper disable All
+using System;
+
+namespace AOT
+{
+    [AttributeUsage(AttributeTargets.Method)]
+    internal class MonoPInvokeCallbackAttribute : Attribute
+    {
+        private Type type;
+        public MonoPInvokeCallbackAttribute(Type t) { type = t; }
```

Comment:
Style-wise, do we want to allow single-line methods to be written like this?

Reply:
As the other reviews said.